internal/controllers/topology/cluster/cluster_controller_test.go

```diff
@@ -768,6 +768,7 @@ func setupTestEnvForIntegrationTests(ns *corev1.Namespace) (func() error, error)
 				WithMachineDeployment(machineDeploymentTopology2).
 				WithVersion("1.22.2").
 				WithControlPlaneReplicas(3).
+				WithControlPlaneNodeDrainTimeout(&metav1.Duration{Duration: time.Second * 10}).
```

Comment:
This test should fail first - then I'll push the fix.

Reply:
This didn't fail properly 🙁 because it's the KCP webhook that objects in this case. I'll update the e2e test.